content/rs/references/rest-api/objects/crdb_task.md

```diff
@@ -0,0 +1,22 @@
+---
+Title: CRDB task object
+linkTitle: crdb_task
+description: An object that represents a CRDB task
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+
+An object that represents a CRDB task.
```

Comment:
```suggestion
An object that represents a CRDB (Active-Active) task.
```
We might want to clarify that CRDB and Active-Active are the same thing in places where we need to use CRDB.

Reply:
Fixed.